src/renderer/actions/validator.ts

```diff
@@ -1,24 +1,59 @@
-import {ValidatorResponse} from "@chainsafe/lodestar-types";
+import {Gwei} from "@chainsafe/lodestar-types";
 import {Action, Dispatch} from "redux";
+
 import {ValidatorActionTypes} from "../constants/action-types";
+import {IValidator} from "../containers/Dashboard/DashboardContainer";
 import {IRootState} from "../reducers";
 import {fromHex} from "../services/utils/bytes";
 
-export const loadValidator = (validator: string) => {
+export interface ILoadValidators {
+    type: typeof ValidatorActionTypes.LOAD_VALIDATORS,
+    payload: Array<IValidator>,
+}
+
+export const loadValidatorsAction = () => {
+    return async (dispatch: Dispatch<Action<unknown>>, getState: () => IRootState): Promise<void> => {
+        const auth = getState().auth;
+        if (auth && auth.account) {
+            const validators = auth.account.getValidators();
+            const validatorArray = validators.map((v) => ({
+                name: auth.account!.name,
+                status: "TODO status",
+                publicKey: v.publicKey.toHexString(),
+                deposit: 30,
```

Comment:
This can also be fetched from node instead of stubbed

Reply:
I am not sure that we even need this info? Maybe we will when we support multiple deposits with values lower than 32 eth but currently this is not used and can remove it

Reply:
You can deposit more than 32 ethers though, but yeah, we can probably remove for now